### ISSUE TITLE
Error: Internal process error (null)

### ISSUE THREAD

**Issue by efade8476** [LedgerHQ/ledger-live-desktop#1140]
[ledgerlive-export-2018.07.15-20.04.34-c7c5c51e1dd0136251485114e6020ab354789a23.log](https://github.com/LedgerHQ/ledger-live-desktop/files/2195696/ledgerlive-export-2018.07.15-20.04.34-c7c5c51e1dd0136251485114e6020ab354789a23.log)
Error was thrown:

```
Error
    at o (file:///Applications/Ledger%20Live.app/Contents/Resources/app.asar/renderer.js:1:25413)
    at a (file:///Applications/Ledger%20Live.app/Contents/Resources/app.asar/renderer.js:1:25600)
    at EventEmitter.l (file:///Applications/Ledger%20Live.app/Contents/Resources/app.asar/renderer.js:1:27016)
    at emitTwo (events.js:130:20)
    at EventEmitter.emit (events.js:213:7)
```

**Comment by gre** [comment]
please try again on release 1.0.2

**Comment by KhalilBellakrid** [comment]
@efade8476 on which OS are you ?

**Comment by gre** [comment]
@KhalilBellakrid  in logs we can see `(Macintosh; Intel Mac OS X 10_11_6)`

**Comment by gre** [comment]
i think this will be fixed in 1.0.4 by #1193
